mep/people/tests/test_admin.py

```diff
@@ -43,6 +44,33 @@ def test_merge_people(self):
         assert mockrequest.session['people_merge_filter'] in \
             ['p=3&filter=foo', 'filter=foo&p=3']
 
+
+    def test_tabulate_queryset(self):
+        fixtures = ['sample_people']
+        person_admin = PersonAdmin(model=Person, admin_site=admin.site)
+        people = Person.objects.order_by('id').all()
+        # test that tabular data matches queryset data
+        for person, person_data in zip(people, person_admin.tabulate_queryset(people)):
+            for field in person_admin.export_fields:
```

Comment:
rather than duplicate the logic in the code, I would suggest spot-checking a couple of fields and a couple of the method properties directly